docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,732 @@
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="ComboBox"
+    description="ComboBox is the combination of a Textfield and an associated Dropdown to let the user filter a list when selecting. ComboBox allows to type the selection, type part of the selection and narrow the results, and select an option from the list."
+    defaultCode={`
+function ComboBoxExample(props) {
+  const PRONOUNS = [
+    'ey / em',
+    'he / him',
+    'ne / nem',
+    'she / her',
+    'they / them',
+    've / ver',
+    'xe / xem',
+    'xie / xem',
+    'zie / zem',
+  ];
+
+  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
+
+  return (
+    <ComboBox
+      accessibilityClearButtonLabel="Clear the current value"
+      accessibilityShowButtonLabel="Show a list of pronoun values for this field"
+      label="Pronouns"
+      id="header"
+      noResultText="No results for your selection"
+      options={options}
+      placeholder="Add your pronouns"
+      helperText="Choose your pronouns to appear on your profile so others know how to refer to you. You can edit or remove these any time."
+    />
+  );
+}
+`}
+  />,
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'options',
+        type: 'Array<{| label: string, value: string, subtext: string |}>',
+        description:
+          'The data for each selection option. See [subtext](#With-subtext) variant to learn more',
+        required: true,
+      },
+      {
+        name: 'inputValue',
+        type: 'string',
+        description:
+          'The user input in ComboBox for controlled components. See [controlled ComboBox](#Controlled-vs-Uncontrolled) variant to learn more.',
+      },
+      {
+        name: 'id',
+        type: 'string',
+        description:
+          'Unique id to identify each ComboBox. Used for [accessibility](#Accessibility) purposes.',
+        required: true,
+      },
+      {
+        name: 'label',
+        type: 'string',
+        description: 'Provide a label to identify the ComboBox field.',
+      },
+      {
+        name: 'disabled',
+        type: 'boolean',
+        defaultValue: false,
+        description:
+          'Set disabled state so ComboBox looks inactive, cannot be interacted with, and actions are not available. If tags are passed, they appeared disabled as well. See [tags](#TYags) variant to learn more.',
+      },
+      {
+        name: 'helperText',
+        type: 'string',
+        description: 'Provides additional information about how to select a ComboBox option.',
+      },
+      {
+        name: 'accessibilityShowButtonLabel',
+        required: true,
+        type: 'string',
+        description: 'Provide feedback when an error on selection occurs.',
+      },
+      {
+        name: 'accessibilityClearButtonLabel',
+        type: 'string',
+        required: true,
+        description: 'Label to describe the icon’s purpose.',
```

Comment:
"Label to describe the clear button's purpose, like 'Clear selected pronouns'"

Reply:
DONE